README.md

```diff
@@ -31,6 +31,62 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
 
 You need to submit Github Token to access Github API.
 
+## Getting Started
+##### Pre-Installation Requirements
+###### Node
+- Download and install latest stable version of [Node](https://nodejs.org/en/download/).
+
+###### MongoDb
+- Download and Install [MongoDB Community Edition](https://docs.mongodb.com/manual/installation/#mongodb-community-edition).
```

Comment:
Same goes for Mongo, what version should we (everybody contributing) should use.